tests/spark/autologging/ml/test_pyspark_ml_autologging_custom_allowlist.py

```diff
@@ -27,3 +28,31 @@ def test_custom_log_model_allowlist(tmpdir):
     }
 
     spark_session.stop()
+
+
+def test_log_model_allowlist_from_url():
+
+    allowlist_file_path = "https://raw.githubusercontent.com/mlflow/mlflow/master/mlflow/pyspark/ml/log_model_allowlist.txt"
+
+    spark_session = (
+        SparkSession.builder.config(
+            "spark.mlflow.pysparkml.autolog.logModelAllowlistFile", allowlist_file_path
+        )
+        .master("local[*]")
+        .getOrCreate()
+    )
+
+    mlflow.pyspark.ml.autolog()
+
+    allowlist = set()
+    builtin_allowlist_file = resource_filename("mlflow.pyspark.ml", "log_model_allowlist.txt")
+    with open(builtin_allowlist_file) as f:
+        for line in f:
+            stripped = line.strip()
+            is_blankline_or_comment = stripped == "" or stripped.startswith("#")
+            if not is_blankline_or_comment:
+                allowlist.add(stripped)
+
+    assert mlflow.pyspark.ml._log_model_allowlist == allowlist
```

Comment:
This will fail when mlflow cut branch and run CI on branch (such as https://github.com/mlflow/mlflow/tree/branch-1.26)